src/Test/Html/Internal/ElmHtml/ToString.elm

```diff
@@ -144,6 +146,9 @@ nodeRecordToString options { tag, children, facts } =
                 |> Just
     in
     case toElementKind tag of
+        InvalidElements ->
+            [ "<!-- invalid element -->" ]
```

Comment:
React and Preact throw errors on invalid element names. I opted for creating a comment node instead.

https://github.com/preactjs/preact-render-to-string/blob/27f340b6e7d77ec7775a49a78d105cad26fa0857/src/index.js#L680

https://github.com/facebook/react/blob/ff6283340a10bb72ad0fb16ca027606a9ea1e67c/packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js#L3651-L3663